src/react-app-env.d.ts

```diff
@@ -1,5 +1,72 @@
+declare namespace NodeJS {
```

Comment:
This is duplicated at the bottom of the file

Reply:
indeed